tests/gold_tests/autest-site/microserver.test.ext

```diff
@@ -33,22 +37,6 @@ def httpObject(self, header, data):
     r["body"] = data
     return r
 
-# addResponse adds customized response with respect to request_header. request_header and response_header are both dictionaries
-
-
-def addResponse(self, filename, testName, request_header, response_header):
-
-    txn = dict()
-    txn["timestamp"] = ""
-    txn["uuid"] = testName
-    txn["request"] = request_header
-    txn["response"] = response_header
-
-    addTransactionToSession(txn, filename)
-    absFilepath = os.path.abspath(filename)
-    self.Setup.CopyAs(absFilepath, self.Variables.DataDir)
-    return
-
```

Comment:
This method was overloaded with a form taking one fewer positional arguments, and it is the override below that is added to the process object.

As far as I can tell, this is unused.